Public/Src/Utilities/UnitTests/Utilities/RelativePathTests.cs

```diff
@@ -40,10 +40,10 @@ public void TryCreate()
             XAssert.IsTrue(RelativePath.TryCreate(st, @"AAA\CCC", out p));
             XAssert.AreEqual(@"AAA\CCC", p.ToString(st));
 
+            XAssert.IsFalse(RelativePath.TryCreate(st, @"..", out p));

```

Comment:
>XAssert.IsFalse(RelativePath.TryCreate(st, @"..", out p)); [](start = 12, length = 58)

Is there a change here? Looks like it was just reordered. Please revert if that was just an unintentional change due to debugging

Reply:
Ahh yea looks like I re-ordered it to do a test, and then didn't move it back in the end. Ill change it back